src/core/icons/fileIcons.ts

```diff
@@ -2876,5 +2876,11 @@ export const fileIcons: FileIcons = {
       name: 'minecraft-fabric',
       fileNames: ['fabric.mod.json'],
     },
+    {
+      name: 'umi',
+      patterns: {
+        ".umirc": FileNamePattern.Ecmascript,
```

Comment:
Today I found out that umi also has its own file icon. That's so cool.

Can you also add one for dumi?

https://github.com/umijs/dumi/blob/7718c044d7ebbdba47e07f96c8f9b027a1ef7f51/src/service/constants.ts#L3


<img width="439" height="723" alt="image" src="https://github.com/user-attachments/assets/4aa5e9ae-064b-4371-9173-3ab9a23c3a7d" />